universal-application-tool-0.0.1/app/auth/UatProfile.java

```diff
@@ -104,7 +106,24 @@ public UatProfileData getProfileData() {
       return CompletableFuture.completedFuture(null);
     }
 
-    return getOwnedApplicantIds()
+    return getAccount()
+        .thenApplyAsync(
+            account ->
+                Stream.concat(
+                        account
+                            .getMemberOfGroup()
+                            .flatMap(tiGroup -> Optional.of(tiGroup.getManagedAccounts().stream()))
+                            .orElse(Stream.of()),
+                        Stream.of(account))
+                    .map(Account::ownedApplicantIds)
+                    .reduce(
+                        ImmutableList.of(),
+                        (one, two) -> {
+                          ImmutableList.Builder<Long> builder = new ImmutableList.Builder<>();
+                          builder.addAll(one);
+                          builder.addAll(two);
+                          return builder.build();
```

Comment:
```suggestion
             return new ImmutableList.Builder<Long>.add(one, two).build();
```

Reply:
That would create an `ImmutableList<ImmutableList<Long>>`, I think.  I could do it all on one line, though - I'll do that.